pkgs/applications/misc/ticker/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "ticker";
+  version = "2.4.0";
```

Comment:
The latest release is [3.0.0](https://github.com/achannarasappa/ticker/releases/tag/v3.0.0).

Reply:
Looks like it was just released! Thanks.